test/AutoDiff/stdlib/callee_differential_not_leaked_in_func_with_loops.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import _Differentiation
+import StdlibUnittest
+import CoreFoundation
+
+// When the original function contains loops, we allocate a context object 
+// on the heap. This context object may store non-trivial objects, such as closures,
+// that need to be freed explicitly, at program exit. This test verifies that the 
+// autodiff runtime destroys and deallocates any such objects.
+
+extension LifetimeTracked: AdditiveArithmetic {
```

Comment:
Can we use the existing Tracked wrappers btw?

Reply:
See comment above.